go.mod

```diff
@@ -245,4 +245,6 @@ replace google.golang.org/protobuf => github.com/lightninglabs/protobuf-go-hex-d
 // automatically, so we need to add it manually.
 replace github.com/golang-migrate/migrate/v4 => github.com/lightninglabs/migrate/v4 v4.18.2-9023d66a-fork-pr-2
 
+replace github.com/lightningnetwork/lnd => github.com/lightningnetwork/lnd v0.19.0-beta
+
```

Comment:
why the replace instead of direct update?

Reply:
Because most of the projects still reference the RC3. If I don't use the replace, `go mod tidy` thinks RC3 is equally good and uses that instead. We could do a PR to each project and bump from RC3 to final to avoid this. But I thought about doing this between RC1 and the final version.

Reply:
gotcha! thanks